draft-ietf-gnap-core-protocol.md

```diff
@@ -3733,9 +3740,40 @@ Values defined by this specification are as follows:
 `"jws"`:
 : Attached JWS payload. See {{attached-jws}}.
 
-
 Additional proofing methods are defined by [a registry TBD](#IANA).
 
+For example, the `httpsig` method can be specified with its parameters as:
+
+
+~~~ json
+{
+    "proof": {
+        "method": "httpsig",
+        "alg": "rsa-pss-sha512",
+        "content-digest-alg": "sha512"
+    }
+}
+~~~
+
+If additional parameters are not required or used for a specific method, the method MAY be passed
+as a string instead of an object. For example, the `httpsig` method with no additional parameters could be sent instead as:
+
+~~~ json
+{
+    "proof": "httpsig"
+}
+~~~
+
+This compact form is equivalent to sending the following object:
+
+~~~ json
+{
+    "proof": {
+        "method": "httpsig"
+    }
+}
+~~~
```

Comment:
```suggestion
as a string instead of an object. For example, the `mtls` method with no additional parameters could be sent instead as:

~~~ json
{
    "proof": "mtls"
}
~~~

This compact form is equivalent to sending the following object:

~~~ json
{
    "proof": {
        "method": "mtls"
    }
}
~~~
```